# Get device_id emergency percentage, return all emergencies if device_id set to "all"
string device_id

---
# response is true if value accepted, otherwise server ignore the request and response false
bool response

# Get device_id vector
string[] device_ids

# Get emergency_percentage vector corresponds to device_id vector
float32[] emergency_percentages
